src/net_processing.cpp

```diff
@@ -329,7 +337,17 @@ class PeerManagerImpl final : public PeerManager
      */
     bool MaybeDiscourageAndDisconnect(CNode& pnode, Peer& peer);
 
-    void ProcessOrphanTx(std::set<uint256>& orphan_work_set) EXCLUSIVE_LOCKS_REQUIRED(cs_main, g_cs_orphans);
+    /**
+     * Reconsider orphan transactions after a parent has been accepted to the mempool.
+     *
+     * @param[in,out]  peer             The peer whose orphan transactions we will reconsider. Generally only one
+     *                                  orphan will be reconsidered on each call of this function. This peer's set
+     *                                  may be added to if accepting an orphan causes its children to be
+     *                                  reconsidered.
```

Comment:
Dropped the `out` which doesn't make sense. Knowing that this may increase the size of the set, not simply reduce it is valuable I think, so have left that text alone.